HPXMLtoOpenStudio/measure.rb

```diff
@@ -2745,12 +2745,29 @@ def get_space_temperature_schedule(model, location, spaces)
     space_values = Geometry.get_temperature_scheduled_space_values(location)
 
     if location == HPXML::LocationOtherHeatedSpace
-      # Create a sensor to get dynamic heating setpoint
-      htg_sch = spaces[HPXML::LocationConditionedSpace].thermalZone.get.thermostatSetpointDualSetpoint.get.heatingSetpointTemperatureSchedule.get
-      sensor_htg_spt = OpenStudio::Model::EnergyManagementSystemSensor.new(model, 'Schedule Value')
-      sensor_htg_spt.setName('htg_spt')
-      sensor_htg_spt.setKeyName(htg_sch.name.to_s)
-      space_values[:temp_min] = sensor_htg_spt.name.to_s
+      if spaces[HPXML::LocationConditionedSpace].thermalZone.get.thermostatSetpointDualSetpoint.is_initialized
+        # Create a sensor to get dynamic heating setpoint
+        htg_sch = spaces[HPXML::LocationConditionedSpace].thermalZone.get.thermostatSetpointDualSetpoint.get.heatingSetpointTemperatureSchedule.get
+        sensor_htg_spt = OpenStudio::Model::EnergyManagementSystemSensor.new(model, 'Schedule Value')
+        sensor_htg_spt.setName('htg_spt')
+        sensor_htg_spt.setKeyName(htg_sch.name.to_s)
+        space_values[:temp_min] = sensor_htg_spt.name.to_s
+      else
+        # No HVAC system; use the average of defaulted heating/cooling setpoints.
+        htg_weekday_setpoints, htg_weekend_setpoints = HVAC.get_default_heating_setpoint(HPXML::HVACControlTypeManual, @eri_version)
+        clg_weekday_setpoints, clg_weekend_setpoints = HVAC.get_default_cooling_setpoint(HPXML::HVACControlTypeManual, @eri_version)
+        if htg_weekday_setpoints.split(', ').uniq.size == 1 && htg_weekend_setpoints.split(', ').uniq.size == 1 && htg_weekday_setpoints.split(', ').uniq == htg_weekend_setpoints.split(', ').uniq
+          default_htg_sp = htg_weekend_setpoints.split(', ').uniq[0].to_f
+        else
+          fail 'Unexpected heating setpoints.'
+        end
+        if clg_weekday_setpoints.split(', ').uniq.size == 1 && clg_weekend_setpoints.split(', ').uniq.size == 1 && clg_weekday_setpoints.split(', ').uniq == clg_weekend_setpoints.split(', ').uniq
+          default_clg_sp = clg_weekend_setpoints.split(', ').uniq[0].to_f
+        else
+          fail 'Unexpected cooling setpoints.'
+        end
+        space_values[:temp_min] = (default_htg_sp + default_clg_sp) / 2.0 # avg-F
+      end
```

Comment:
It doesn't make sense to use the htg setpoint in the `if` and the average of htg/clg setpoints in the `else`. Don't blindly copy what was being done for natural ventilation, the code needs to be adjusted for the situation.

That said, I don't really remember why the original implementation is like this. We assign the minimum temperature of [68F, per our documentation](https://openstudio-hpxml.readthedocs.io/en/latest/workflow_inputs.html#hpxml-locations), in `Geometry.get_temperature_scheduled_space_values`, but then we override it w/ the heating setpoint. So the code doesn't match the documentation -- either the documentation needs to be updated or we shouldn't be overriding it. It looks like we started overriding it in [this PR](https://github.com/NREL/OpenStudio-HPXML/pull/400), but I don't remember why. @yzhou601, do you remember what issue that PR is trying to address?

Finally, this still doesn't address the primary issue, which is that the zone temperature is being used in the calculation below. Look at `sensor_ia` below.